Python/libraries/recognizers-date-time/recognizers_date_time/date_time/chinese/date_extractor_config.py

```diff
@@ -111,23 +171,73 @@ def duration_extractor(self) -> DateTimeExtractor:
     def strict_relative_regex(self) -> Pattern:
         return None
 
-    @property
-    def range_connector_symbol_regex(self) -> Pattern:
-        return self._range_connector_symbol_regex
-
     @property
     def utility_configuration(self) -> DateTimeUtilityConfiguration:
         return None
 
+    @property
+    def week_day_end(self) -> Pattern:
+        pass
+
     def __init__(self):
+        self._datetime_period_unit_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.DateTimePeriodUnitRegex
+        )
+        self._after_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.AfterRegex
+        )
+        self._before_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.BeforeRegex
+        )
+        self._date_unit_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.UnitRegex
+        )
+        self._next_prefix_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.NextPrefixRegex
+        )
+        self._last_prefix_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.LastPrefixRegex
+        )
+        self._this_prefix_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.ThisPrefixRegex
+        )
+        self._date_year_in_chinese_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.DateYearInChineseRegex
+        )
+        self._zero_to_nine_integer_regex_chinese = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.ZeroToNineIntegerRegexChs
+        )
+        self._relative_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.RelativeRegex
+        )
+        self._year_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.YearRegex
+        )
+        self._month_num_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.MonthNumRegex
+        )
+        self._day_regex_num_in_chinese = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.DayRegexNumInChinese
+        )
+        self._date_day_regex_in_chinese = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.DateDayRegexInChinese
+        )
+        self._day_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.DayRegex
+        )
+        self._month_regex = RegExpUtility.get_safe_reg_exp(
+            ChineseDateTime.MonthRegex
+        )
+        self._number_parser = BaseNumberParser(
+            ChineseNumberParserConfiguration()
```

Comment:
Why number_parser it is initialized this way? Maybe we could change this to implement something as below:

    def __init__(self, config: ChineseNumberParserConfiguration):
        self._number_parser = config.number_parser

Reply:
We don't have config here, why add it?

Reply:
Nice catch, I'll change this now